### PR TITLE
Update clear-access-levels.md

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
Added complete example

### CODE REVIEW

File: docs/sdks/clear-access-levels.md [hunk 1]
```diff
@@ -10,26 +10,63 @@ import TabItem from '@theme/TabItem';
...
-    await client.clearDatabaseAccessLevel(keyid, '_system')
-}
-catch(e){
-    console.log("Clearing Access Level for Database Failed: ",e)
-}
-try{
-    await client.clearStreamAccessLevel(keyid, '_system', "c8globals."+streamName)
+const jsc8 = require("jsc8");
 
+// Email and Password to authenticate client instance
```
Comment (jamesm-macrometa) [inline docs/sdks/clear-access-levels.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/sdks/clear-access-levels.md [hunk 2]
```diff
@@ -10,26 +10,63 @@ import TabItem from '@theme/TabItem';
...
+const streamName = "testStream";
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+// Or use one of the following authentication methods and remove the commenting.
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/sdks/clear-access-levels.md]:
```suggestion
// Create an authenticated instance with an API key.
```

**Review by jamesm-macrometa — APPROVED** [review]
Minor comments, looks good!

**Comment by jamesm-macrometa** [comment]
@diana-macrometa Want me to go ahead and merge this one, or should I wait for your review?  The content is very similar to the last few of @edgargarciamacrometa's PRs

**Review by diana-macrometa — APPROVED** [review]
G2g once you address James' suggestions.

Btw, you only need one approval to merge.

**Pull request merged**